tests/apps/core/test_templates_base.py

```diff
@@ -0,0 +1,52 @@
+"""
+Test suite for the Open Graph for course pages
+"""
+import re
+
+from django.test.utils import override_settings
+
+from cms.test_utils.testcases import CMSTestCase
+
+from richie.apps.core.helpers import create_i18n_page
+
+
+class TemplatesBaseCMSTestCase(CMSTestCase):
+    """Testing the base.html template"""
+
+    def test_templates_base_meta_og_image(self):
```

Comment:
I think this test should be `test_templates_richie_homepage_meta_og_image` in a file named `test_templates_richie_homepage.py`

Reply:
Renamed